.gitignore

```diff
@@ -2,3 +2,12 @@ serialbox
 __pycache__
 .vscode
 **.gt_cache/
+*.dat
```

Comment:
This can cause issues with the ice.dat and land.dat files in the seaice/fortran scheme.

Reply:
Good catch, removing this